ranger/core/main.py

```diff
@@ -407,8 +407,14 @@ def load_custom_commands(*paths):
         except OSError:
             LOG.debug('Unable to access plugin directory: %s', plugindir)
         else:
-            plugins = [p[:-3] for p in plugin_files
-                       if p.endswith('.py') and not p.startswith('_')]
+            plugins = []
+            for path in plugin_files:
+                if not path.startswith('_'):
+                    if path.endswith('.py'):
+                        plugins.append(path[:-3])
```

Comment:
What's this magical `-3` about?

Reply:
It removes the trailing `.py`, since that gives the name of the python module. I just added a comment for that.